### ISSUE TITLE
Pressing on backspace with only one character in input value deletes the last tag

### ISSUE THREAD

**Issue by xiurobert** [xiurobert/dtags-input#3]
Unfortunately for some strange reason if the input value consists of only one character, pressing on the delete button will cause the last tag to be deleted

https://github.com/xiurobert/dtags-input/blob/0167ad9dc9b10743dc4f07cad4c9e00e216b46b3/src/tag_input.js#L51-L55

**Comment by ngmhprogramming** [comment]
if you backspace on a single character doesn't it become an empty string i.e. ""

**Comment by xiurobert** [comment]
Yes, that it correct. But it is deleting the tag AND the single character. If there is a single character, the tag should not be deleted
